rpm/libcap.spec

```diff
@@ -1,12 +1,11 @@
 Name: libcap
-Version: 2.25
+Version: 2.69
 Release: 1
 Summary: Library for getting and setting POSIX.1e capabilities
 Source: %{name}-%{version}.tar.bz2
-Patch0: libcap-2.25-buildflags.patch
 
 URL: https://github.com/sailfishos/libcap
-License: LGPLv2+
+License: BSD-3-Clause OR GPLv2
```

Comment:
Should the first one be just BSD now? JB#50383 for earlier adjustments. Getting a feeling Fedora probably changed their conventions on these.

Reply:
Changed